### ISSUE TITLE
Make a home page to be redirected to after login

### ISSUE THREAD

**Issue by ahmeddrawy** [ahmeddrawy/my-udemy#9]
- [ ] make a static home page to view courses

- [ ] use `/api/courses` endpoint to fetch courses from database and view them dynamically on home page

**Comment by emma-r-slight** [comment]
I'd like to give this a go..... :)

**Comment by ahmeddrawy** [comment]
@emma-r-slight thank you, please make it dynamic and any meta data you want to add to course model or user model please let me know, I'll assign you

**Comment by emma-r-slight** [comment]
Do you have any preference for a template(view) engine? express-handlebars, pug, dots.....?  What information do you want to be listed from the course routes? Do you want just a list with name and tags, do you want to include an external link to the course?

**Comment by ahmeddrawy** [comment]
I was thinking about `ejs` but never used it, also I was planning to learn React 
As a first step we can show their names and tags maybe we can edit the model too to add photos or another metadata
What do you think?

**Comment by emma-r-slight** [comment]
Ok, I have used express handlebars before, which is quite a good stepping stone to react.  It separates parts of the page into components similar to React. 

For the courses I was looking at the Udemy page, it looks like there's an image( at this point I don't think it needs to be saved in the db, just use a random image from unsplash.com), Title, name of creater and then topic(tags).  

Do you have a problem if I add some tests using Jest?

**Comment by ahmeddrawy** [comment]
>Do you have a problem if I add some tests using Jest?

No, I know jest and was planning to have tests but i don't know how until now and will we apply unit or integration testing, what do you think ?

**Comment by emma-r-slight** [comment]
At this point unit testing, I haven't used Nosql databases before (only SQL Databases) so wanted to write tests for the 'courses' db and api route. 
This is a great [article](https://medium.com/javascript-in-plain-english/how-i-setup-unit-test-for-mongodb-using-jest-mongoose-103b772ee164) about testing 'login' if you were interested.  It shows how to setup a Mongodb.config file for testing
